test/erlcloud_s3_test_data.hrl

```diff
@@ -0,0 +1,507 @@
+%%%-------------------------------------------------------------------
+%%% @author <Kirill Starikov kstarikov@alertlogic.com>
+%%% @copyright (C) 2016, <Alert Logic>
+%%% @doc
+%%%
+%%% @end
+%%% Created : 15. Feb 2016 22:21
+%%%-------------------------------------------------------------------
+-author("<Kirill Starikov kstarikov@alertlogic.com>").
+
+-define(S3_BUCKET_EVENT_XML_CONFIG,
+    {xmlElement,'NotificationConfiguration','NotificationConfiguration',[],
```

Comment:
I think it is a back pain to change this test data if needed. Having it in a raw xml will help greatly.